### PR TITLE
Fix file format ref: #402

### PR DESCRIPTION
**Pull request by appleboy**
According to #402, I update `cp` document with single description and single command line.

### CODE REVIEW

**Comment by appleboy** [comment]
screen shot 2015-12-28 at 10 12 13 pm

File: pages/common/cp.md [hunk 1]
```diff
@@ -8,14 +8,20 @@
...
 
-`cp {{/path/to/original}} ../{{/path/to/copy}}`
+`cp {{/path/to/original}} ../{{path/to/copy}}`
 
-- Copy directories recursive using the option -r. Optionally showing files as they are copied.
+- Copy directories recursive using the option -r
 
 `cp -r {{/path/to/original}} {{/path/to/copy}}`
+
+- Optionally showing files as they are copied
```
Comment (waldyrious) [inline pages/common/cp.md]: "Optionally showing" --> "Show"

File: pages/common/cp.md [hunk 2]
```diff
@@ -8,14 +8,20 @@
...
+
 `cp -vr {{/path/to/original}} {{/path/to/copy}}`
 
-- Make a copy of a file adding and extension or changing an extension
+- Make a copy of a file adding and extension
```
Comment (waldyrious) [inline pages/common/cp.md]: "and" --> "an"

File: pages/common/cp.md [hunk 3]
```diff
@@ -8,14 +8,20 @@
...
+
+`cp {{file.html}}{,.backup}`
+
+- Changing file extension
```
Comment (waldyrious) [inline pages/common/cp.md]: I'd suggest "Make a copy of a file, changing the extension".

**Comment by waldyrious** [comment]
Nice one! Please take a look at the comments I've added inline.

**Comment by appleboy** [comment]
@waldyrious Done, please review again. Thanks :+1:

**Comment by waldyrious** [comment]
perfect :ok_hand: :)

**Comment by felixonmars** [comment]
:+1:

**Comment by appleboy** [comment]
Finally screenshot 

screen shot 2015-12-29 at 1 40 53 pm

**Comment by igorshubovych** [comment]
LGTM

**Comment by appleboy** [comment]
rebase master branch again. @igorshubovych Could you merge this branch?

**Pull request merged**